lib/services/servermanagement/generate.cmd

```diff
@@ -0,0 +1,17 @@
+::

```

Comment:
We do not use this. We use gulp. Please follow the steps in the documentation. https://github.com/Azure/azure-sdk-for-node/blob/master/Documentation/RegeneratingCode.md
We are using the master branch and not the autorest branch. Please refer to the onboarding doc for service team over [here](https://github.com/Azure/adx-documentation-pr/blob/master/clis/azure-cli/guide.md#hydra-specs-pr----node-sdk----xplat-cli)